test/units/module_utils/basic/test_exit_json.py

```diff
@@ -158,7 +158,7 @@ def test_fail_json_removes_values(self, am, capfd, return_val, expected, monkeyp
         monkeypatch.setattr(warnings, '_global_deprecations', [])
         expected['failed'] = True
         with pytest.raises(SystemExit):
```

Comment:
What's the expected message in this exception?

Reply:
Line 164 is testing this already.